services/venice-controller/src/main/java/com/linkedin/venice/pushmonitor/PushMonitorUtils.java

```diff
@@ -285,11 +285,14 @@ public static ExecutionStatusWithDetails getDaVinciPartitionLevelPushStatusAndDe
         if (instanceStatus.equals(PushStatusStoreReader.InstanceStatus.BOOTSTRAPPING)) {
           // Don't count bootstrapping instance status report.
           totalReplicaCount--;
-          LOGGER.info(
-              "Skipping ingestion status report from bootstrapping node: {} for topic: {}, partition: {}",
-              entry.getKey().toString(),
-              topicName,
-              partitionId);
+          skipLogCount++;
```

Comment:
Why do we log only 10 entries?

Reply:
What don't we use `RedundantExceptionFilter` here?
Maybe we should rename it to something like `LogRateLimiter`, since it can be used for the regular logging as well.

Reply:
Sure will use RedundantExceptionFilter